bootstrap.py

```diff
@@ -12,7 +12,7 @@
 import io
 
 
-minimal_egg_info = textwrap.dedent("""
+minimal_egg_info = textwrap.dedent(u"""
```

Comment:
Use `from __future__ import unicode_literals` if unicode literals are required... which I don't think they are here. Scrap this change.